accel-pppd/radius/dm_coa.c

```diff
@@ -158,6 +179,12 @@ static void coa_request(struct radius_pd_t *rpd)
 		rad_packet_print(rpd->dm_coa_req, NULL, log_ppp_info2);
 	}
 
+	/* Process RADIUS attributes in CoA request message and update the
+	corresponding session with the new values. This is done before the triton
+	event is triggered so the redis module can read and publish the new values
+	from the session data structure */
+	handle_coa_attributes(rpd);
+
```

Comment:
For [`ipoe`](https://github.com/xebd/accel-ppp/blob/a77f9e071f084918bbb805c2509aec0f47fcf95b/accel-pppd/ctrl/ipoe/ipoe.c#L2417-L2500) the radius attributes in the CoA messages are handled within the module

Reply:
Probably we should at least check the `ev->ses->ctrl->type` ([similar as in ipoe.c](https://github.com/xebd/accel-ppp/blob/a77f9e071f084918bbb805c2509aec0f47fcf95b/accel-pppd/ctrl/ipoe/ipoe.c#L2425))